### ISSUE TITLE
tsd install should install every definitions listed in `tsd.json`

### ISSUE THREAD

**Issue by mvidailhet** [DefinitelyTyped/tsd#116]
With npm and bower you just need to specify "install" to install everything listed in their configuration file. I think tsd should behave the same way to avoid confusing the user.

I searched a long time before finding the right command to do what I wanted:
```
tsd reinstall -s
```

**Comment by dduugg** [comment]
:+1:

**Comment by sandorfr** [comment]
Yes ! That would be so much easier. In addition it will reduce learning curve for new users.

**Comment by eps90** [comment]
:+1:

**Comment by Diullei** [comment]
I started it here https://github.com/Diullei/tsd/commit/4671696899fb089340d652c10d41189693cbed84

**Comment by un33k** [comment]
+!
